src/screens/send/components/fee-advanced/fee-advanced.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
```

Comment:
as this component is related to blockchain that are using GAS, rename this as `gas-fee-advanced`